src/components/Cards/BookCard/styles.js

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components'
+
+export const Container = styled.section`
+  max-width: 178px;
+  width: 100%;
+  min-height: 487px;
+  height: auto;
+`
+
+export const Image = styled.img`
+  margin-bottom: var(--spacing-1);
+`
+
+export const LineContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+
+  margin-bottom: var(--spacing-2);
+`
+
+export const Title = styled.h2`
```

Comment:
Podemos usar uma tag `h1` aqui, na última atualização do HTML é recomendado que todo title de um componente seja h1, ele não quebra mais o SEO

[Pode ver mais sobre aqui](https://www.greenlanemarketing.com/resources/articles/seo-101-seo-and-multiple-h1-tags/)

![image](https://user-images.githubusercontent.com/59345698/218222567-38450d56-5acb-47dd-b661-1581d2e56300.png)